message_definitions/v1.0/development.xml

```diff
@@ -38,5 +47,16 @@
       <field type="float" name="raw_press" units="hPa">Raw differential pressure. NaN for value unknown/not supplied.</field>
       <field type="uint8_t" name="flags" enum="AIRSPEED_SENSOR_FLAGS">Airspeed sensor flags.</field>
     </message>
+    <message id="420" name="RADIO_RC_CHANNELS">
+      <description>Radio channels. Supports up to 32 channels. Channel values are in centered 13 bit format. Range is [-4096,4096], center is 0. Conversion to PWM is x * 5/32 + 1500.  
```

Comment:
should mention that this is an set of RC channel inputs for a flight controller, it does not replace the RC_CHANNELS output message

Reply:
Thx, will be done